RealLifeGTAV/presence.ts

```diff
@@ -0,0 +1,73 @@
+const presence = new Presence({
+  clientId: "760863245224640555"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```

Comment:
merge consts

Reply:
Have I merged it? Everything here confuses me xD

Reply:
You didn't just push commits to your repo master branch.

Reply:
Merge consts:
const var1 = obj,
var2 = obj;

Reply:
What should I do with it? (I've never done anything like that before :/)

Reply:
**_Pending_** what have I done now xD

Reply:
```javascript
const presence = new Presence({
  clientId: "760863245224640555"
}),
 browsingStamp = Math.floor(Date.now() / 1000);
```